wagtail/images/views/images.py

```diff
@@ -34,19 +34,31 @@
 
 
 class BaseListingView(TemplateView):
+    ORDERING_OPTIONS = {
+        "-created_at": "Newest",
```

Comment:
nitpick: the labels need to be translatable

Reply:
Thanks @zerolab! I've updated it.